server/api/endpoints/faq.py

```diff
@@ -43,3 +44,43 @@ def create(data: FaqCreate = Body(...)) -> Any:
             status_code=HTTPStatus.INTERNAL_SERVER_ERROR,
             detail=f"Internal server error: {str(e)}",
         )
+
+
+@router.put("/{faq_id}", response_model=FaqUpdated, status_code=HTTPStatus.CREATED)
+def update(*, faq_id: UUID, item_in: FaqUpdate) -> FaqUpdated:
+    try:
+        faq = faq_crud.get(faq_id)
```

Comment:
This is a very long try.

Not sure what can all go wrong whilst inside the try, but it ain't pretty.

What are you trying to catch here? The IntegrityError?

The put the try just before that statement.